package.json

```diff
@@ -31,28 +31,29 @@
     "main": "can-stache-bindings"
   },
   "dependencies": {
+    "can-attribute-encoder": "^0.3.0",
```

Comment:
I released this as > `0.1.0` so that we could publish patches for docs and things.